### ISSUE TITLE
Search query error when using `withCount`

### ISSUE THREAD

**Issue by OzanKurt** [yajra/laravel-datatables#1643]
@yajra 

I just faced a weird problem while using search for a counted query. 

```php
        $query = ProgramSession::select([
            'program_sessions.*',
        ])->withCount([
            'attendances',
        ]);
```

If I were to add that `withCount` function the aggregate count query fails. Since the `where`s are not being added to the query inside, instead they are being added the the main query.

**Comment by Skintillion** [comment]
What error are you getting please provide more details

**Comment by rudashi** [comment]
I have the same issue. If we use `withCount` the aggregate Count is always 0.
For some reasons, the query use bindings from `withCount`

![Code](https://i.imgur.com/4QV0i07.jpg)

![DebugBar](https://i.imgur.com/NIPElTJ.jpg)

**Comment by Skintillion** [comment]
You can also put the counts in as attributes instead, then you can skip the "with" entirely.

**Comment by yajra** [comment]
Can you please check again using v8.5.1, this be fixed by #1737 and #1730. Thanks!
